plexapi/client.py

```diff
@@ -548,7 +540,7 @@ def setStreams(self, audioStreamID=None, subtitleStreamID=None, videoStreamID=No
 
     # -------------------
     # Timeline Commands
-    def timeline(self, wait=1):
```

Comment:
I want the default argument to be the same as before. Leave this 1 and just call it with 0 where you need it,